### PR TITLE
Add ApiScan to the compliance pipeline

### PR DESCRIPTION
**Pull request by jgoshi**

### CODE REVIEW

File: azure-pipelines/compliance.yml [hunk 1]
```diff
@@ -62,6 +62,16 @@ jobs:
     - task: CredScan@3
       inputs:
         scanFolder: '$(Build.SourcesDirectory)'
+    - task: APIScan@2
+      displayName: 'Run APIScan'
+      inputs:
+        softwareFolder: '$(Build.ArtifactStagingDirectory)/drop'
+        softwareName: 'Visual Studio Tools for Unreal Engine'
+        softwareVersionNum: '2.4'
```
Comment (hnrqbaggio) [inline azure-pipelines/compliance.yml:70]: Do you think there's a way for us to keep this in sync with the value in the `.uplugin` file, which controls the submission to the Unreal Marketplace? Or do we even care? I'm not quite sure how this is used for the task itself.

**Review by hnrqbaggio — DISMISSED** [review]

File: azure-pipelines/compliance.yml (hunk 1 above)
Comment (jgoshi) [reply azure-pipelines/compliance.yml:78]: I'm not sure either. And since I don't know how to keep this in sync, we can just leave it like this for now.

Comment (jgoshi) [reply azure-pipelines/compliance.yml:87]: Actually seems like that value is not used.

**Review by hnrqbaggio — APPROVED** [review]
LGTM.

**Pull request merged**